tedpermission/src/main/java/com/gun0912/tedpermission/TedPermissionActivity.java

```diff
@@ -256,7 +257,7 @@ public void onSaveInstanceState(Bundle outState) {
     public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions,
                                            @NonNull int[] grantResults) {
 
-        ArrayList<String> deniedPermissions = TedPermissionBase.getDeniedPermissions(this, permissions);
+        List<String> deniedPermissions = TedPermissionBase.getDeniedPermissions(this, permissions);
```

Comment:
Is there a reason to change this?
It is a local variable and there is no connection point with the outside.

Reply:
It was just base on `program to interfaces, not implementations`.

Although this request changed the listener used by the` ArrayList` to `List`, it was consistent under the same principle.

Reply:
It was just base on `program to interfaces, not implementations`.

Although this request changed the listener used by the` ArrayList` to `List`, it was consistent under the same principle.